VMBackup/main/hostsnapshotter.py

```diff
@@ -231,8 +231,11 @@ def get_snapshot_info(self, responseBody):
                         self.logger.log("Converting the creationTime string received in UTC format to UTC Ticks")
                         delta = creationTimeObj - epochTime
                         timestamp = self.get_total_seconds(delta)
-                        creationTimeUTCTicks = str(int(timestamp * 1000)) 
-                        ddSnapshotIdentifierInfo = HostSnapshotObjects.DDSnapshotIdentifier(creationTimeUTCTicks , snapshot_info['ddSnapshotIdentifier']['id'], snapshot_info['ddSnapshotIdentifier']['token'])
+                        creationTimeUTCTicks = str(int(timestamp * 1000))
+                        instantAccessDurationMinutes = None
+                        if 'instantAccessDurationMinutes' in snapshot_info['ddSnapshotIdentifier']:
```

Comment:
can we use the constant instantAccessDurationMinutes defined in common.py in lines 236 and 237 also while accessing the property